kbcstorage/tables.py

```diff
@@ -42,11 +39,28 @@ def list(self, include=None):
         Raises:
             requests.HTTPError: If the API request fails.
         """
-        headers = {'X-StorageApi-Token': self.token}
 
         url = '{}/tables'.format(self.base_url)
         params = {'include': ','.join(include)}
-        return self.get(url, headers=headers, params=params)
+        return self._get(url, params=params).json()
+
+    def list_bucket(self, bucket_id, include=None):
+        """
+        List all tables in a bucket.
+
+        Args:
+            bucket_id (str): Id of the bucket
+            include (list): Properties to list (attributes, columns)
+        Returns:
+            response_body: The parsed json from the HTTP response.
+
+        Raises:
+            requests.HTTPError: If the API request fails.
+        """
+
+        url = '{}/{}/tables'.format(self.base_url, bucket_id)
+        params = {'include': ','.join(include)}
+        return self._get(url, params=params).json()
```

Comment:
This API call is already implemented here https://github.com/keboola/sapi-python-client/blob/25633efff914ecd82913e0af923e5e4dfafabf94/kbcstorage/buckets.py#L39

which I think is the correct place for it. Even though in the [docs](http://docs.keboola.apiary.io/#reference/tables/tables-in-bucket) it's in the `tables` section, the url indicates the buckets endpoint.

Reply:
+1